templates/terraform/decoders/global_neg_fetch_endpoint.go.erb

```diff
@@ -0,0 +1,36 @@
+config := meta.(*Config)
+
+selfLink, ok := res["selfLink"]
+if !ok {
+	return res, nil
+}
+url := fmt.Sprintf("%s/listNetworkEndpoints", selfLink)
+
+project, err := getProject(d, config)
+if err != nil {
+	return nil, err
+}
+endpointsRes, err := sendRequest(config, "POST", project, url, nil)
+if err != nil {
+	return nil, handleNotFoundError(err, d, fmt.Sprintf("ComputeGlobalNetworkEndpoint %q", d.Id()))
+}
+
+items, ok := endpointsRes["items"]
+if !ok {
+	return res, nil
+}
+
+itemArray := items.([]interface{})
+if len(itemArray) == 0 {
+	return res, nil
+}
+// Global endpoint groups are unable to have more than 1 endpoint
```

Comment:
if they start allowing this, we'll have to go and remove this check. I'm fine with keeping it, but if we do have this check, I'd also want to leave a note in the docs under the resource saying currently only one is supported.